coltune_analyze.py

```diff
@@ -44,7 +44,6 @@ def load_single_result(file_name, collective):
         return load_omb_single_result(file_name, collective)
 
 def load_imb_single_result(file_name, collective):
-    from string import atoi, atof, split
```

Comment:
Can you squash this commit with the previous one? Otherwise the previous commit will not work.

Reply:
Will squash with other commits.